.github/ISSUE_TEMPLATE/coblocks-and-or-gutenberg-update-github-tracking-issue-template.md

```diff
@@ -0,0 +1,39 @@
+---
+name: 'CoBlocks and or Gutenberg update GitHub Tracking Issue template'
+about: 'Template for Gutenberg/Coblocks plugin update process'' GitHub tracking issue,
+  more: '
+title: Update Gutenberg to v<version> and CoBlocks to v<version>
+labels: ''
+assignees: ''
+
+---
+
+We have installed Gutenberg `v<version>` and CoBlocks `v<version>` to edge sites for early feedback. 
+
+Feedback and bugs and the final activation of the plugins on wpcom will be tracked here, for anyone interested.
+
+Testing post: p7DVsv-8Oh-p2
+
+### Process
+
+* [x] Install version 8.3.0-rc.1. (D44703-code)
```

Comment:
should they be un-checked by default? (not sure if the template works differently :p)

Reply:
Ah no, sorry, they should be!

Reply:
Let me know if you think any steps should be added/removed. The idea is not to be a one-size-fits-all, but more of an initial structure that can be used to bootstrap the issue, as some updates might require us to add new steps or more text, etc. Not all updates will be equal, I guess.